token/program-2022/src/offchain.rs

```diff
@@ -1,6 +1,7 @@
 //! Offchain helper for fetching required accounts to build instructions
 
 pub use spl_transfer_hook_interface::offchain::{AccountDataResult, AccountFetchError};
+#[allow(deprecated)]
```

Comment:
nit: rather than having one huge `allow` on the whole import statement, it might be clearer to only add it on the import for `resolve_extra_account_metas`.

But I just noticed that gets fixed in the next PR, so never mind!

Reply:
No worries, I'll add it here anyway!